### ISSUE TITLE
"lifetime" Variable and "population" Array

### ISSUE THREAD

**Issue by rkr8** [rkr8/Krabll#5]
## `lifetime` Variable and `population` Array

Variable `lifetime`:
In my opinion, we don't need the variable `lifetime`. We could just simply delete every krabll, whose input `alive` is false. Afterwards, we could simply add an `if`-statement, which checks, if less than 10 krablls are alive.

`population` array:
We should rather use a list instead of an array for our `population`, so that we are able to remove specific elements.

**Comment by dromedarCase** [comment]
I agree that we should rather use a list instead of an array for the `population`. Still I tend to keep the lifetime variable and not to delete dead Krablls, for following reasons:
- Keeping the Krablls might give us the option to save backups of single stages of the Krabll's development and give us the ability to retry a generation with another `elite size`.
- The elite Krablls may be sorted based on their lifetime to give us a ranking.

**Comment by rkr8** [comment]
Ok, we could set `alive` to `false`, instead of deleting the Krabll. But in this case, we don't need the variable `lifetime` too.

**Comment by dromedarCase** [comment]
I'm waiting on #krabll
